src/platform/plugins/shared/dashboard/server/content_management/v2/cm_services.ts

```diff
@@ -37,7 +37,7 @@ export const serviceDefinition: ServicesDefinition = {
       ...serviceDefinitionV1?.create?.in,
       data: {
         schema: dashboardAttributesSchema,
-        up: (data: DashboardCrudTypes['CreateIn']['data']) => attributesTov3(data),
+        up: (data: DashboardCrudTypes['CreateIn']['data']) => attributesTov3(data, [], () => []),
```

Comment:
Why do we pass a blank references array here?

Reply:
Ah, those are now optional parameters. The content management `up` transforms do not supply `references` so we only transform the attributes. Fixed in ff8e17a77ec737fbbcc160b1636904dc9f49660f.